docs/metecho-how-to.md

```diff
@@ -0,0 +1,150 @@
+# Steps to Guide a Team Through Metecho
+
+## Step 0 - Log in with GitHub
+- Create a GitHub account, if you don’t yet have one.
+- Contact an admin for the repository (e.g. Nonprofit Success Pack) on GitHub and ensure that you and anyone you’d like to work with are collaborators. 
+
+If you want to create a project, skip down to [Create a Project](#create-a-project).
+
+
+## Step 1 - Select a Project
+**Project** - *(e.g. Nonprofit Success Pack) Projects contain all the work being done. They are equivalent to GitHub repositories.*
+
+- Select the Project you want to contribute to. 
+- If you do not see the Project you want to work on: 
+  - Confirm that you are logged into the correct Metecho account.
+  - Contact an admin for the repository on GitHub and ensure that you are a collaborator. 
+  - Select Re-sync Projects in Metecho.
```

Comment:
I think adding screenshots would be helpful in cases where we mention the UI directly. A running instance is available at https://metecho-dev.herokuapp.com/